test/SlowTests/Issues/RavenDB-19096.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Indexes;
+using Raven.Server.NotificationCenter.Notifications;
+using Raven.Server.NotificationCenter.Notifications.Details;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Sparrow.Server.Collections;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_19096 : RavenTestBase
+{
+    public RavenDB_19096(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    [RavenFact(RavenTestCategory.None)]
+    public async Task AssertIndexNamesAreAddedToCpuCreditsExhaustionNotification()
+    {
+        DoNotReuseServer();
+        
+        using (var store = GetDocumentStore())
+        {
+            var db = await GetDatabase(store.Database);
+
+            var notificationsQueue = new AsyncQueue<DynamicJsonValue>();
+
+            using (db.NotificationCenter.TrackActions(notificationsQueue, null))
+            {
+                var index1 = new DummyIndex();
+                var index2 = new OtherIndex();
```

Comment:
Do you need to create those indexes?

Reply:
No, it's enough to add some strings directly into warning. I changed it